Assets/Scripts/ConnectionManagement/ConnectionMethod.cs

```diff
@@ -1,47 +1,42 @@
-using System;
 using System.Threading.Tasks;
-using Unity.BossRoom.UnityServices.Lobbies;
+using Unity.BossRoom.UnityServices.Sessions;
 using Unity.BossRoom.Utils;
 using Unity.Netcode.Transports.UTP;
-using Unity.Networking.Transport.Relay;
 using Unity.Services.Authentication;
 using Unity.Services.Core;
-using Unity.Services.Relay;
-using Unity.Services.Relay.Models;
 using UnityEngine;
 
 namespace Unity.BossRoom.ConnectionManagement
 {
     /// <summary>
-    /// ConnectionMethod contains all setup needed to setup NGO to be ready to start a connection, either host or client side.
-    /// Please override this abstract class to add a new transport or way of connecting.
+    ///     ConnectionMethod contains all setup needed to setup NGO to be ready to start a connection, either host or client
```

Comment:
Seems like we may have different Rider settings, since that's not being picked up on my Rider locally. I would suggest to remove these whitespace fixes in this PR since it's adding some noise to the PR too. We can look into any sort of formatting in a follow-up PR.